src/libmongoc/src/mongoc/mongoc-client-side-encryption.c

```diff
@@ -2324,7 +2352,8 @@ mongoc_client_encryption_remove_key_alternate_name (
       BSON_ASSERT (bson_iter_init (&iter, &local_reply));
 
       if (bson_iter_find_descendant (&iter, "value.keyAltNames", &iter)) {
-         if (BSON_ITER_HOLDS_ARRAY (&iter) && bson_iter_recurse (&iter, &iter)) {
+         if (BSON_ITER_HOLDS_ARRAY (&iter) &&
+             bson_iter_recurse (&iter, &iter)) {
```

Comment:
Just noticing this line. Is it kosher to use the same `bson_iter_t` for `bson_iter_recurse`? We've always used separate variables in PHPC.

Reply:
I believe so. There are 7 existing instances of `bson_iter_recurse (&iter, &iter)` (exactly as written) in the repo. Given `bson_iter_t` behaves as a view (non-owning), there is no risk of state invalidation. Otherwise, `bson_iter_recurse` documentation should be updated to explicitly forbid self-recursion.